docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -583,3 +583,32 @@ function sitenow_page_attachments(array &$attachments) {
     $attachments['#attached']['library'][] = 'sitenow/admin-overrides';
   }
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function sitenow_toolbar() {
+
+  $url = Url::fromUri('https://docs.sitenow.uiowa.edu');
+
+  $items = [];
+  $items['support'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'link',
+      '#url' => $url,
+      '#title' => 'SiteNow Help',
+      '#options' => [
+        'attributes' => [
+          'title' => t('Opens help documentation in a new window'),
+          'target' => '_blank',
+        ],
+      ],
+    ],
+    // Devel is weighted 999. This can be altered later
+    // if additional toolbar options are needed.
+    '#weight' => 1100,
```

Comment:
I don't this is needed. Setting weights is slippy slope similar to z-index. I think it is fine if this is before the user, devel, and tour buttons.

Reply:
If near-last on the toolbar is desired, the tour module has specific class alongside toolbar-tab to target and adds a float:right bit of css.

Reply:
Looking at it with fresh eyes, I don't think the weight is needed--the class option is a good one to keep in mind for future reference though.

Removed the weight.